demo/demoApp.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+ 
```

Comment:
please go over this file and make sure all indentations are correct as it seems some lines got misaligned.